Resources/Prototypes/Entities/Objects/Specific/Medical/handheld_crew_monitor.yml

```diff
@@ -31,15 +31,71 @@
   - type: StationLimitedNetwork
   - type: StaticPrice
     price: 500
+
+- type: entity
+  name: cmo's handheld crew monitor
+  parent: HandheldCrewMonitor
+  suffix: DO NOT MAP
+  id: CMOHandheldCrewMonitor
+  components:
   - type: Tag
     tags:
     - HighRiskItem
   - type: StealTarget
     stealGroup: HandheldCrewMonitor
 
 - type: entity
-  id: HandheldCrewMonitorEmpty
+  id: CMOHandheldCrewMonitorEmpty
+  parent: CMOHandheldCrewMonitor
+  suffix: Empty
+  components:
+  - type: ItemSlots
+    slots:
+      cell_slot:
+        name: power-cell-slot-component-slot-name-default
+
+- type: entity
+  name: paramedic handheld crew monitor
   parent: HandheldCrewMonitor
+  id: ParamedicHandheldCrewMonitor
+  description: A paramedic-specific hand-held crew monitor displaying the status of suit sensors. It only shows crew in a critical state or already dead.
+  components:
+  - type: Sprite
+    sprite: Objects/Specific/Medical/handheldcrewmonitor.rsi
```

Comment:
inherited so remove this line

Reply:
the sprite changed and the description is different...?

Reply:
the parent `BaseHandheldCrewMonitor` already has the path written so you need just to change state for children